### ISSUE TITLE
cif-router service is erroring out when re-starting container

### ISSUE THREAD

**Issue by dwdixon** [sfinlon/cif-docker#3]
When stopping the cifv3 container and starting it back up again the cif-router service is not starting up as it should be for some reason, the observed error is as follows:

```
2019-08-29 18:58:35,008 DEBG 'cif-router' stderr output:
2019-08-29 18:58:35,008 - CRITICAL - cif.router[305][MainThread] - /tmp/cif_router.pid already exists, exiting
2019-08-29 18:58:35,030 INFO exited: cif-router (exit status 0; not expected)
2019-08-29 18:58:35,030 DEBG received SIGCLD indicating a child quit
2019-08-29 18:58:36,032 INFO gave up: cif-router entered FATAL state, too many start retries too quickly
```

**Comment by dwdixon** [comment]
I tested the fix for the supervisorctl issue #1  as we discussed and it seems to have worked
but after stopping the container and restarting it, then when cif-router is dead (this issue) I attempted to manually start it up  via ```supervisorctl start cif-router``` it starts it up fine…but then when testing a cif query it still throws a BadStatusLine error even with cif-router running, I also tried to manually restart the cif-http service to see if that was part of the problem but that didn't help and I still got the BadStatusLine error:

```
root@container:/var/log/supervisor# supervisorctl status
cif-httpd                        RUNNING   pid 12, uptime 0:04:59
cif-router                       RUNNING   pid 88, uptime 0:01:19
csirtg-smrt                      RUNNING   pid 13, uptime 0:04:59
root@container:/var/log/supervisor# su cif -c "whoami && cif -d -q www.google.com"
cif
2019-08-29 21:29:32,205 - DEBUG - urllib3.connectionpool[205][MainThread] - Starting new HTTP connection (1): localhost:5000
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/cifsdk/client/client.py", line 259, in main
    rv = cli.search(filters)
  File "/usr/local/lib/python2.7/dist-packages/cifsdk/client/plugin.py", line 20, in search
    return self.indicators_search(data)
  File "/usr/local/lib/python2.7/dist-packages/cifsdk/client/http.py", line 201, in indicators_search
    rv = self._get('/search', params=filters)
  File "/usr/local/lib/python2.7/dist-packages/cifsdk/client/http.py", line 88, in _get
    resp = self.session.get(uri, params=params, verify=self.verify_ssl, timeout=self.timeout)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 546, in get
    return self.request('GET', url, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/adapters.py", line 498, in send
    raise ConnectionError(err, request=request)
ConnectionError: ('Connection aborted.', BadStatusLine("''",))
2019-08-29 21:30:03,629 - ERROR - cifsdk.client.client[270][MainThread] - ('Connection aborted.', BadStatusLine("''",))
```

**Comment by sfinlon** [comment]
Arrow changed funcionality in 0.15.2, pinned version in csirtg-smrt and csirtg-indicator
